examples/mining_ar.py

```diff
@@ -0,0 +1,128 @@
+          'once in the dataset. Now let\'s see the results with '
+          f'{COLOR_CODES["yellow"]}minsup=0.5{COLOR_CODES["default"]} and '
+          f'{COLOR_CODES["yellow"]}minconf=0.5{COLOR_CODES["default"]}.\n')
+    algo.execute(minsup=0.5, minconf=0.5)
+    print_ars(algo.get_ars())
+    print('\nNow you can see that the number of association rules have decreased '
+          'significantly. This happened due to minsup being set to 0.5, which basically '
+          'means that association rule needs to are present in 50% of all transactions '
+          "in order to be included in the result. Unfortunately, you can't get support "
+          'for each individual association rule with Desbordante.\n'
+          '\nA typical approach to controlling the algorithm is to employ "usefulness", '
+          'which is defined as confidence * support. In the last example, we set up min '
+          '"usefulness" = 0.5 * 0.5 = 0.25. \n\nNow let\'s try with '
+          f'{COLOR_CODES["green"]}minsup=0.7{COLOR_CODES["default"]}, '
+          f'{COLOR_CODES["green"]}minconf=0.7{COLOR_CODES["default"]} '
+          f'and {COLOR_CODES["bold_yellow"]}"usefulness"=0.49{COLOR_CODES["default"]}\n')
+    algo.execute(minsup=0.7, minconf=0.7)
+    print_ars(algo.get_ars())
+    print('\nSo, now the total number of returned association rules is only four. '
+          'We reduced the amount of "noisy" information in our output. However, '
+          'this is just a single example of how you can evaluate the usefulness of data. '
+          'You are free to play with these parameters to see how it changes things. '
+          'Eventually, you will find out what fits your dataset and your task best.')
+
+
+def scenario_singular():
+    algo = desbordante.ar.algorithms.Default()
+    algo.load_data(table=(TABLE_SINGULAR, ',', True), input_format='tabular')
+    algo.execute()
+    table = pandas.read_csv(TABLE_SINGULAR, header=None, index_col=0)
+
+    print("\nFor the second example, let's take a dataset with the same receipts "
+          'from the same supermarket with input_format="singular". '
+          'This is a two-column format, where the second column denotes '
+          'item and the first one denotes the order of items it belongs to.\n')
+    print_table(table)
+    print('\nThis format is just a different table representation. Desbordante '
+          'is perfectly capable of the discovered rules from such tables, '
+          'and the found rules are the same objects as in the previous '
+          'examples (i.e., they possess the same methods and fields).\n')
+    print('In this example, we show you how to list all of the unique items in the dataset.')
+    print(f'For this you can use {COLOR_CODES["bold_blue"]}'
+          f'get_itemnames{COLOR_CODES["default"]} method:\n')
+    print_itemnames(algo.get_itemnames())
+    print('\nNow you have all items used in this dataset, '
+          'so you can filter it and then print it.\n')
```

Comment:
Depending on the context, you could write "allowing you to filter them and then print individual entries"